### PR TITLE
add user photo to header

### PR DESCRIPTION
**Pull request by jesusantguerrero**
- Add user photo

### CODE REVIEW

File: src/components/gw-user-options.vue [hunk 1]
```diff
@@ -124,6 +124,8 @@ export default {
 
         img {
             width: 100%;
+            min-width: 100%;
+            min-height: 100%;
```
Comment (SparoHawk) [inline src/components/gw-user-options.vue]: These are not necessary.
The width of the image is already defined as 100%.
The reason for not having a height is to allow correct image proportions. Though I can see the issue with that here.

Also, in CSS there is no reason to use `min-width` or `min-height` as 100%. Just use `width` and `height`.

I think what we need to do is add `overflow: hidden;` to the `.profile-image` parent class.

**Review by SparoHawk — CHANGES_REQUESTED** [review]

File: src/components/gw-user-options.vue (hunk 1 above)
Comment (jesusantguerrero) [reply src/components/gw-user-options.vue]: It seems that flex can overwrite/ignore width and height to render the element in the available space. setting `min-height` and `min-width` stops this behavior.

But let me see how it behaves with `overflow:hidden`

Comment (SparoHawk) [reply src/components/gw-user-options.vue]: @jesusantguerrero Please check the comment if you haven't.

Comment (jesusantguerrero) [reply src/components/gw-user-options.vue]: > @jesusantguerrero Please check the comment if you haven't.

@SparoHawk  done

**Review by SparoHawk — APPROVED** [review]

**Pull request merged**